### ISSUE TITLE
Write-Influx issue with first parameter only work through Pipe and not as a parameter

### ISSUE THREAD

**Issue by Tonic8** [markwragg/PowerShell-Influx#18]
Hello

- we cannot use the command line 
`write-influx -inputObject $data -Database "Data" -Server "Server"`

- the command cannot even be used in splatting 
```
$splat = @{
  InputObject = $Data
  Database = "Data"
  Server = "Server"
}
write-influx @splat
```
- We can mix the concepts but the InputObject must be passed by the Pipe
```
$splat = @{
  Database = "Data"
  Server = "Server"
}
$Data |write-influx @splat

or

$Data | write-influx -Database "Data" - Server "Server"
```

When doing that the error message is:
```
Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Collections.Hashtable".
At c:\test\_modules\Influx\Public\Write-Influx.ps1:101 char:13
+                  $Metrics = $InputObject.Metrics
+                  ~~~~~~~~~~~~~~~~~~~~~
       + CategoryInfo             : Invalid Argument: (:) [], RuntimeException
       + FullyQualifiedErrorId : ConvertToFinalInvalidCastException
```
Thanks
Tonic8

**Comment by markwragg** [comment]
You're right, it was only expected that the `-InputObject` parameter would be used via the pipeline as it only exists to support input from any cmdlet that generates an object of type "metric". That being said, this is fixable by just adding a ForEach loop, which i'll do.

**Comment by markwragg** [comment]
This should be fixed as of version 1.0.93. Please update the module and let me know if it works for you.

**Comment by Tonic8** [comment]
hello

test, ok!!! :)

all working fine.
